src/site/tests/support/index.js

```diff
@@ -0,0 +1,94 @@
+import { readFileSync } from 'fs';
+import path from 'path';
+import { JSDOM } from 'jsdom';
+import liquid from 'tinyliquid';
+import registerFilters from '../../filters/liquid.js';
+import createRedirects from '../../stages/build/plugins/rewrite-va-domains.js';
+import rewriteAWSUrls from '../../stages/build/plugins/rewrite-cms-aws-urls.js';
+import modifyDom from '../../stages/build/plugins/modify-dom';
+
+registerFilters();
+
+const updateHTML = files => {
+  const options = {
+    buildtype: process.env.BUILDTYPE, // what should i set this to?
```

Comment:
We can probably just hardcode this to `ENVIRONMENTS.VAGOVDEV`

`import ENVIRONMENTS from 'site/constants/environments';`